### DISCUSSION TITLE
Portfolio Bug

### DISCUSSION THREAD

**Discussion by arcturiano** [BitCannaGlobal/WeedWallet#85] [General]
When going into Portfolio and selecting Simple mode, I see Atmoner's Portfolio as title, although BCNA balance,  address and transactions are mine. 

![PortfolioBug](https://github.com/BitCannaGlobal/WeedWallet/assets/94879750/b27b2ae0-0388-4f22-9c04-ad9fbb941179)

**Comment by TJ-BitCanna** [comment]
Correct :)

I will close this one though, since it has been reported here https://github.com/BitCannaGlobal/WeedWallet/discussions/58 by @kjincgroup already.

**Comment by arcturiano** [comment]
Had not seen that it had been reported my bad!

**Reply by TJ-BitCanna** [reply]
No worries, that's why I check :)

And rather double than nothing.
